test/observable_test.dart

```diff
@@ -23,15 +24,15 @@ void main() {
   });
 
   test('Raw observables', () {
-    final x = ObservableValue(1000);
+    final x = ObservableValue(mobxContext, 1000);
```

Comment:
We will also need some tests that will provide custom context and verify its being used correctly.

Reply:
I think that will apply to actions, observables, reactions